src/index.ts

```diff
@@ -109,6 +109,8 @@ export interface GossipsubOpts extends GossipsubOptsSpec, PubSubInit {
   asyncValidation: boolean
   /** Do not throw `InsufficientPeers` error if publishing to zero peers */
   allowPublishToZeroPeers: boolean
+  /** Do not throw `PublishError.Duplicate` if publishing duplicate messages */
+  ignoreDuplicateMessages: boolean
```

Comment:
```suggestion
  allowPublishDuplicateMessages: boolean
```

Reply:
Disagree with this for reasons mentioned here https://github.com/ChainSafe/js-libp2p-gossipsub/pull/404#issuecomment-1437401486